tartiflette/language/parsers/libgraphqlparser/parser.py

```diff
@@ -30,7 +30,11 @@
 # TODO: use importlib.resource in Python 3.7
 _LIBGRAPHQLPARSER_DIR = os.path.join(os.path.dirname(__file__), "cffi")
 try:
-    _LIB = _FFI.dlopen(f"{_LIBGRAPHQLPARSER_DIR}/libgraphqlparser.so")
+    dir_from_env = os.environ.get("LIBGRAPHQLPARSER_DIR")
```

Comment:
I'll prefer to have 
```
_LIBGRAPHQLPARSER_DIR = os.environ.get("LIBGRAPHQLPARSER_DIR", os.path.join(os.path.dirname(__file__), "cffi"))
```
The rest of the logic could stay the same. no need for the "dir_from_env" var.